### ISSUE TITLE
Rally - Improve focus support

### ISSUE THREAD

**Issue by perclasson** [material-components/material-components-flutter-gallery#491]
- Right now in Rally for desktop the focus starts at the alerts, instead of the navigation. Make sure that the focus order makes more sense.

**Comment by perclasson** [comment]
Right now these two issues are not easily supported and as such I consider this ticket blocked.

- We currently lack support in the framework to have a custom focus traversal policy: https://github.com/flutter/flutter/issues/47349
- To allow the overview graph to access the back button, we need to be able to have a traversal policy that goes outside of the Navigator scope, see issue https://github.com/flutter/flutter/issues/47344.

**Comment by perclasson** [comment]
With https://github.com/flutter/flutter/issues/47349 fixed we can now improve Focus support.
